spec/intl.html

```diff
@@ -157,6 +157,7 @@ <h1>InitializeDateTimeFormat ( _dateTimeFormat_, _locales_, _options_ )</h1>
         1. <del>If _dateTimeFormat_.[[Hour]] is *undefined*, then</del>
           1. <del>Set _dateTimeFormat_.[[HourCycle]] to *undefined*.</del>
           1. <del>Let _pattern_ be _bestFormat_.[[pattern]].</del>
+          1. <del>Let _rangePatterns_ be _bestFormat_.[[rangePatterns]].</del>
```

Comment:
The [upstream proposal](https://tc39.es/proposal-intl-DateTimeFormat-formatRange/) uses `[[pattern]]` and `[[Pattern]]` inconsistently, and doesn't appear to have them as separate slots; only `Pattern` and `RangePatterns` are described in the [section](https://tc39.es/proposal-intl-DateTimeFormat-formatRange/#sec-properties-of-intl-datetimeformat-instances); same for `[[rangePatterns]]`. Not sure whether to suggest fixing here or keeping this way for compatibility until fixed there, but heads up.

Reply:
My understanding was that `[[Pattern]]` and `[[RangePatterns]]` were internal slots of DTF objects, but `[[patterns]]` and `[[rangePatterns]]` were keys for these pattern records.